common/artifacts/999_wc_artifacts_other_consumables.txt

```diff
@@ -0,0 +1,29 @@
+# Food, potions, alcohol
+
+#Alcohol
+halaani_whiskey_keg = {
+	quality = 1
+	flags = { alcohol }
+	picture = "GFX_halaani_whiskey_keg"
+}
```

Comment:
You should ban gifting the kegs to those who can't drink it.

Reply:
Not resolved.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/103140083-d76e5580-46fb-11eb-8b0a-d2e68b5e9f98.png)

</details>

Reply:
right, forgot to push. fixed now